src/test/java/com/puppycrawl/tools/checkstyle/utils/ModuleReflectionUtilsTest.java

```diff
@@ -140,6 +140,7 @@ protected InvalidNonAutomaticBeanClass() {
         public abstract void method();
     }
 
+    /** @noinspection unused */
```

Comment:
please make javadoc for this class, to explain reason of it. Suppressions for unused looks weird.